.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+
+python:
+  - 3.6
+
+install:
+  - pip install --upgrade -r requirements.in
```

Comment:
Is there a reason to make this three different calls all with `--upgrade`?  Furthermore, it is good practice to create a `virtualenv` and not use the system python environment eventhough you only require a single env.

Reply:
Well, according to https://docs.travis-ci.com/user/languages/python/#travis-ci-uses-isolated-virtualenvs I am running in a dedicated freshly created `virtualenv` and *not* in the system env. That being said, I figured that I use the simple way until we hit any issues to keep our build setup as simple as possible until we hit any issues